### ISSUE TITLE
Provide a default user and use that as the node author in the test

### ISSUE THREAD

**Issue by Berdir** [larowlan/default_content#36]
The test is currently a bit weird, it has an incomplete embedded reference for the uid and expects that to be 0 then, which isn't happening.

**Comment by andypost** [comment]
yep, faced with with too in custom entity

```
InvalidArgumentException: Field author_uid is unknown. in Drupal\Core\Entity\ContentEntityBase->getTranslatedField() (line 370 of core/lib/Drupal/Core/Entity/ContentEntityBase.php)
```

but my entity is not translatable

**Comment by larowlan** [comment]
Fixed in #38
